src/avitab/AviTab.cpp

```diff
@@ -35,7 +35,7 @@ AviTab::AviTab(std::shared_ptr<Environment> environment):
     if (env->getConfig()->getBool("/AviTab/loadNavData")) {
         env->loadNavWorldInBackground();
     }
-    chartService = std::make_shared<apis::ChartService>(env->getProgramPath() + "/Navigraph/");
+    chartService = std::make_shared<apis::ChartService>(env->getProgramPath());

```

Comment:
As I don't have access to the Navigraph build, I can't check whether I've broken anything. I don't think I have, but...

Reply:
Checked, works.